aws/resource_aws_api_gateway_gateway_response.go

```diff
@@ -0,0 +1,152 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/apigateway"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsApiGatewayGatewayResponse() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsApiGatewayGatewayResponseCreate,
+		Read:   resourceAwsApiGatewayGatewayResponseRead,
+		Update: resourceAwsApiGatewayGatewayResponseCreate,
+		Delete: resourceAwsApiGatewayGatewayResponseDelete,
+
+		Timeouts: &schema.ResourceTimeout{
+			Delete: schema.DefaultTimeout(5 * time.Minute),
+		},
```

Comment:
Just wanted to make it all flexible. Your point seems legit, will remove the related stuff!